entrypoint.sh

```diff
@@ -1,27 +1,33 @@
 #!/bin/bash
 
-INPUT_CONFIG_PATH="$1"
-CONFIG=""
+set -eu -o pipefail
+
+# Assume the $GITHUB_WORKSPACE is a safe directory
+# https://github.blog/2022-04-12-git-security-vulnerability-announced/
+git config --global --add safe.directory "$GITHUB_WORKSPACE"
+
+extra_args=""
 
 # check if a custom config have been provided
-if [ -f "$GITHUB_WORKSPACE/$INPUT_CONFIG_PATH" ]; then
-  CONFIG=" --config-path=$GITHUB_WORKSPACE/$INPUT_CONFIG_PATH"
+if [ ! -z "$1" ]; then
```

Comment:
caused by `set -u` in line 3 this will fail when no custom config has been provided. (Maybe I just did not understand the rationale behind dropping the default config)

Reply:
In GitHub Actions this argument is supplied as the empty string, so no unbound variable errors. It's a bit confusing as written though, so I'll update to first check the argument count.